pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -259,13 +265,13 @@ func (ks *scaler) applyScale(ctx context.Context, pa *pav1alpha1.PodAutoscaler,
 	return nil
 }
 
-// Scale attempts to scale the given PA's target reference to the desired scale.
-func (ks *scaler) Scale(ctx context.Context, pa *pav1alpha1.PodAutoscaler, sks *nv1a1.ServerlessService, desiredScale int32) (int32, error) {
+// Scale decides the desired scale and creates PodScalable required to scale the given PA's target reference to the desired scale.
+func (ks *scaler) Scale(ctx context.Context, pa *pav1alpha1.PodAutoscaler, sks *nv1a1.ServerlessService, desiredScale int32) (int32, *pav1alpha1.PodScalable, error) {
```

Comment:
Should we rename this to something like `CalculateScale(...)` since it no longer does the actual application of the new scale?

Reply:
or just `Calculate`?